.github/workflows/triage-issues.yaml

```diff
@@ -0,0 +1,29 @@
+name: Issue Management
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  triage_or_add_to_project:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check if the issue creator is a maintainer
+        id: check_maintainer
+        run: |
+            echo "::set-output name=is_maintainer::$(curl -s -H 'Authorization: token ${{ secrets.GITHUB_TOKEN }}' \
```

Comment:
This was the only way I could figure out how to check whether someone is a maintainer of the repo, but it's pretty ugly so if there's a better way to do this check I would love to use it.

Reply:
How about this action? 
https://github.com/prince-chrismc/check-actor-permissions-action#conditionally-continue

Your implementation has no dependencies and is a minimal implementation

Reply:
Resolving in favor of using `gh` api